server

```diff
@@ -83,6 +83,292 @@ query_default_version() {
   DEFAULT_SCYLLA_VERSION=$(echo $DEFAULT_SCYLLA_VERSION_RAW | sed -e "s/.*version\":\"\(.*\)\".*/\1/g")
 }
 
+query_series_latest_patch() {
+  local series=$1
+  
+  # Try to get the latest patch version for a specific series from GitHub releases
+  # This is more dynamic than hardcoded values
+  local github_releases=$(curl -s "https://api.github.com/repos/scylladb/scylladb/releases?per_page=50" 2>/dev/null)
```

Comment:
I think this part is not relevant for out case..
the https://api.github.com/repos/scylladb/scylladb/releases?per_page=50 always gets an empty list [] as we are not using GitHub “Releases”, https://github.com/scylladb/scylladb/releases - "There aren’t any releases here"
we are using only tags

Reply:
oh you are right, removed this and the relevant refs